scripts/gen-test-account/dist/main.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+var __importDefault = (this && this.__importDefault) || function (mod) {
```

Comment:
This looks generated and probably committed by mistake? Should we adjust the .gitignore?